### DISCUSSION TITLE
Service Status

### DISCUSSION THREAD

**Discussion by lmangani** [deta/cloud-docs#321] [Ideas]
Is there a service status page for the Deta Base, Micros, Drive APIs available?

For instance, today I'm running into many 504's from openresty and lots of missed inserts in Base, which seem unrelated to the code. In general, being able to know the service status would help greatly to avoid troubleshooting in void when the infrastructure is flapping or during outages, etc.

Thanks for your time!

**Comment by lemonyte** [comment]
~~I've created a rudimentary implementation of this, available at https://service-status.deta.dev . It has the endpoints `/micro`, `/base`, and `/drive`. Its still a WIP, the repo is [here](https://github.com/LemonPi314/deta-status).~~

## Update
There is now a completed status page at https://status.deta.dev
